packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -474,17 +474,29 @@ export class ExpressApp {
       });
     });
 
-    /* THIS WAS NEVED ENABLED YET NOW 2020-04-07
     router.post('/v4/txproposals/', (req, res) => {
-      req.body.txpVersion = 4;
       getServerWithAuth(req, res, server => {
+        req.body.txpVersion = 4;
```

Comment:
this will change to: `useSchnoor=true`

then in `createTx`: if  (coin == bch && useSchoor)  => signingMethd=='schorr' on txp